### PR TITLE
Update download testing

### PR DESCRIPTION
**Pull request by rotosonic**
Why: present downloads test is too long and captures too many spikes
How: Update test to generically add sizes and connections
Test: run locally but update testplan to point to a staging server and test connected by ethernet to modem and verify bandwidths.

### CODE REVIEW

File: controllers/TestPlanController.js [hunk 1]
```diff
@@ -87,12 +87,14 @@ class TestPlanController {
         if (global.hasAddressIpv6) {
             testPlan.hasIPv6 = true;
             testPlan.baseUrlIPv6 = '[' + global.AddressIpv6 + ']:' + global.webPort;
+            testPlan.baseUrlIPv6NoPort = '[' + global.AddressIpv6 + ']';
```
Comment (kemarc) [inline controllers/TestPlanController.js]: so this is constructing in IPv6 address?

File: public/examples/download/downloadApp.js [hunk 2]
```diff
@@ -328,14 +325,17 @@
                 myChart.setOption(option, true);
         }
 
+      urls.length=0;
+      var baseUrl = (version === 'IPv6') ? testPlan.baseUrlIPv6NoPort : testPlan.baseUrlIPv4NoPort;
+      for (var i = 0; i < ports.length; i++) {
```
Comment (kemarc) [inline public/examples/download/downloadApp.js]: Is a double for loop needed?

File: controllers/TestPlanController.js (hunk 1 above)
Comment (rotosonic) [inline controllers/TestPlanController.js]: yes

File: public/examples/download/downloadApp.js (hunk 2 above)
Comment (rotosonic) [inline public/examples/download/downloadApp.js]: yes need to loop thru ports and create 6 url for each port

**Comment by rotosonic** [comment]
opened a new pr

**Pull request closed without merging**